apps/aecontract/src/aect_contracts.erl

```diff
@@ -196,6 +205,13 @@ is_legal_version_at_protocol_(call, #{vm := VMVersion}, Protocol) ->
                                    (VMVersion =:= ?VM_FATE_SOPHIA_1)
                                    ) ->
             true;
+        P                     when (P >= ?IRIS_PROTOCOL_VSN)
+                                   and
+                                   (
+                                   (VMVersion =:= ?VM_FATE_SOPHIA_1) or
```

Comment:
This is completely redundant. It is already covered by the case above...

Reply:
fixed